sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -20,10 +20,10 @@ from form documents. It includes the following main functionalities:
 Install the Azure Form Recognizer client library for Python with [pip][pip]:
 
 ```bash
-pip install azure-ai-formrecognizer
+pip install azure-ai-formrecognizer --pre
 ```
 
-> Note: This version of the client library supports the v2.0 version of the Form Recognizer service
+> Note: This version of the client library defaults to the v2.1-preview version of the service
```

Comment:
can be saved for later, but thoughts on adding an API version -> SDK version chart a la TA https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/textanalytics/azure-ai-textanalytics#install-the-package